### ISSUE TITLE
Component: MultiSelect aria required children

### ISSUE THREAD

**Issue by mristov** [primefaces/primeng#15001]
### Describe the bug

![image](https://github.com/primefaces/primeng/assets/150306240/f1884801-7c0b-4de0-9c31-445504811910)

### Environment

PrimeNg 17.10

### Angular version

17.0.5

### PrimeNG version

17.10

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.1

**Comment by TaneliTuomola** [comment]
Each child of a listbox should have a role of option so fixing #15153 might fix this one also.

**Comment by cetincakiroglu** [comment]
Fixed in https://github.com/primefaces/primeng/pull/15153
